recipes/scevan/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: scevan
+  version: 1.0.0  # Replace with the desired version
+
+source:
+  url: https://github.com/AntonioDeFalco/SCEVAN/archive/refs/tags/1.0.0.tar.gz  # Replace with the GitHub URL of the release
+  sha256: cc14b6b80019e06181bc69ab6f54bde9878b3b525645f925626bfed28df655fd  # Generate the checksum using 'sha256sum' command
+
+build:
+  number: 0
+  noarch: generic  # Specify that the package is noarch
+  run_exports: '{{ pin_compatible("SCEVAN", max_pin="x.x") }}'
+  rpaths:
+    - lib/R/lib
+    - lib
+  script: "$R -e \"install.packages(c('Rcpp', 'remotes', 'devtools'), lib='$PREFIX/lib/R/library', repos='https://cran.rstudio.com/')\""
+
+requirements:
+  build:
```

Comment:
I think this should be `host`, not `build`.
R, as Python, is an interpreted language, so these dependencies should be listed in `requirements>host`

Reply:
will edit, thanks!